database/bfgd/postgres/postgres.go

```diff
@@ -1127,7 +1130,8 @@ func (p *pgdb) BtcTransactionBroadcastRequestGetNext(ctx context.Context, onlyNe
 		if err := rows.Scan(&serializedTx); err != nil {
 			return nil, err
 		}
-
+		// LIMIT 1 above so linter needs to shut up
+		// nolint:staticcheck
```

Comment:
I think the linter's issue here is partially valid, whilst this code works it can be simplified:

```go
	var serializedTx []byte
	err := p.db.QueryRowContext(ctx, querySql).Scan(&serializedTx)
	if err != nil {
		return nil, fmt.Errorf("could not get next btc_transaction_broadcast_request: %w", err)
	}

	return nil, nil
```

Reply:
I like this.